iina/Base.lproj/PrefGeneralViewController.xib

```diff
@@ -566,17 +566,51 @@
                         <binding destination="IBP-Mz-Maa" name="value" keyPath="values.playlistShowMetadataInMusicMode" id="nxW-hX-9co"/>
                     </connections>
                 </button>
+                <button verticalHuggingPriority="750" translatesAutoresizingMaskIntoConstraints="NO" id="H91-rO-GsT">
+                    <rect key="frame" x="118" y="7" width="269" height="18"/>
+                    <buttonCell key="cell" type="check" title="Repeat when media is opened manually:" bezelStyle="regularSquare" imagePosition="left" state="on" inset="2" id="jok-kc-HjX">
```

Comment:
Under the `History` section it says: `Otherwise only files that opened manually…`
To be consistent with terms `media` should be `file`.
The wording is a bit odd because `Repeat` is used twice. And `Repeat` is not used in the OSD or the menus. To be consistent `Loop` should be used. Maybe something like:
`When a file is opened manually:` `Loop playlist`

Also, the new text must be added to the English translation file.

Reply:
Sorry I kept forgetting adding the strings